biolink-model.yaml

```diff
@@ -6048,6 +6048,13 @@ slots:
     is_a: association slot
     range: disease or phenotypic feature
 
+  allelic requirement:
+    description: >-
+      the gene variant's inheritance pattern that is associated with the disease or
```

Comment:
```suggestion
       A relationship that specifies the necessary allele configuration of a particular gene or variant required for the expression of a phenotype in a specific patient or instance. This edge is used to link genetic variants to their relevance in the context of the patient's phenotype, based on the inheritance pattern and genetic context.  
```

Reply:
I'm unsure of this description change:
* to me, it sounds like it's describing a predicate ("A relationship...", "This edge is used...")
* EBI gene2pheno uses this on gene-disease edges. So I think this association slot should be okay to use for variant - DiseaseOrPheno relationships AND gene - DiseaseOrPheno.

What do you think of this adjusted description? (multi-line to avoid Github warnings)

```
the allele configuration of a particular gene or variant required for the expression of a disease or phenotype in a specific patient or instance. This edge property may be used by associations between Genes or SequenceVariants and DiseaseOrPhenotypicFeatures to provide the inheritance pattern and genetic context of the relationship.
```